package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "dex-react",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "start": "NODE_ENV=development webpack-dashboard -- webpack-dev-server --mode development --hot",
+    "build": "rimraf ./dist && NODE_ENV=production webpack --mode production",
+    "eslint": "eslint . --ext .ts,.tsx,.js",
+    "eslint-fix": "eslint --fix . --ext .ts,.tsx,.js",
+    "stats": "NODE_ENV=production webpack --mode production --env.stats --profile --json > stats.json && webpack-bundle-analyzer ./stats.json ./dist"
+  },
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@hot-loader/react-dom": "^16.8.6",
+    "apollo-boost": "^0.4.3",
+    "apollo-link-ws": "^1.0.18",
+    "graphql": "^14.4.2",
+    "react": "^16.8.6",
+    "react-apollo": "^2.5.8",
+    "react-dom": "^16.8.6",
+    "react-hot-loader": "^4.12.10",
+    "react-router-dom": "^5.0.1",
+    "styled-components": "^4.3.2",
+    "subscriptions-transport-ws": "^0.9.16",
+    "web3": "^2.0.0-alpha"
```

Comment:
What the decision about web3, we did you go for `2.0.0-alpha` and not for example`1.0.0-beta.55`?

Reply:
They are the same, beta.37 became 1.2.0, beta.55 -- 2.0.0-alpha